ConscryptMode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Conscrypt Mode
+group: User Guide
+order: 2
+redirect_from: "/activity-lifecycle/"
```

Comment:
Utzcoz was saying we should merge this with other existing documentation as a sub section. I'm thinking we could just put this as a sub category inside of http://robolectric.org/configuring/ since this is basically just configuring robolectric's security providers. For now, I think it's safe to move it to the bottom of the configuring.md file:

https://github.com/robolectric/robolectric.github.io/blob/master/configuring.md